src/master-playlist-controller.js

```diff
@@ -361,6 +370,32 @@ export class MasterPlaylistController extends videojs.EventTarget {
         } else {
           addSeekableRange();
         }
+
+        buffered = this.tech_.buffered();
```

Comment:
I think it would be good to pull out this section of calculations to a separate function as this event handler is getting quite gnarly